common-services.yml

```diff
@@ -50,4 +48,26 @@ services:
         max-buffer-size: 100m
       driver: json-file
 
+  exporter_postgres:
+    image: audius/exporter-postgres:v0.10.1
+    restart: always
+
+  exporter_elasticsearch:
+    image: quay.io/prometheuscommunity/elasticsearch-exporter:v1.3.0
+    restart: always
+
+  exporter_redis:
+    image: audius/exporter-redis:v1.44.0-alpine
+    restart: always
 
+  exporter_linux:
+    image: quay.io/prometheus/node-exporter:v1.4.0
+    container_name: exporter_linux
+    network_mode: host
+    command: --path.rootfs=/host
+    pid: host
+    restart: unless-stopped
+    volumes:
+      - '/:/host:ro,rslave'
+    cap_add:
+      - SYS_TIME
```

Comment:
lots of custom configs here with no apparent explanation - mind documenting :)

Reply:
I added line 63 to this file (https://github.com/prometheus/node_exporter/blob/98a40bd712eaeb1fd3ee60ca37e4091c503d8e1c/README.md#docker), but I don't think this comment should show up in this file since it was mainly copy-paste from the exporter's repo on how to run this container with docker-compose.

Reply:
in that case, worth clarifying that this is vendored code at least, in addition to that link so readers undersatnd

Reply:
It's already "documented" that this is a vendored dockerized exporter when you look at line 66 since this is coming from the `prometheus` org:

```
image: quay.io/prometheus/node-exporter:v1.4.0
```

It feels weird that we have this comment for `exporter_linux` but no matching comments for the other exporters, which are to varying degrees also vendored.

I'm still -0 on adding both of these comments, but am happy to leave them in if they help. :)

Reply:
yah please leave them in
anyone looking at this file will be confused about the configs, good to clarify